pkg/oc/cli/cmd/set/env.go

```diff
@@ -77,11 +79,10 @@ var (
 )
 
 type EnvOptions struct {
```

Comment:
That particular command I left out because I want to switch to the upstream version, but for that we need a rebase.